components/brex/actions/create-card/create-card.mjs

```diff
@@ -0,0 +1,24 @@
+import common from "./common.mjs";
+import brexApp from "../../brex.app.mjs";
+
+export default {
+  ...common,
+  name: "Create Card",
+  description: "Creates a new card. [See the docs here](https://developer.brex.com/openapi/team_api/#operation/createCard).",
+  key: "brex-create-card",
+  version: "0.0.1",
+  type: "action",
+  props: {
```

Comment:
Looks like `reason` and `lock_after_date` props should be optional, according to their docs

Reply:
Hey @dannyroosevelt. Its already optional, please check `common.mjs` on this same folder

Reply:
Hm I see that. This isn't rendering as Optional in the UI, so I'll look into that.